com.unity.uiwidgets/Runtime/engine/UIWidgetsPanel.cs

```diff
@@ -264,6 +270,14 @@ void CollectGarbageOnDemand()
             return true;
         }
 #endif
+#if UNITY_IOS || UNITY_STANDALONE_OSX || UNITY_EDITOR_OSX
+        void InitializeOpenGLCore() {
+            OpenGLCoreUtil.RenderTextureCreateFailureWorkaround();
```

Comment:
I don't know why call RenderTextureCreateFailureWorkaround before init?

Reply:
yes, I think we can call `OpenGLCoreUtil.Init` first here since the two calls don't affect each other. But as we will eventually remove the call of `RenderTextureCreateFailureWorkaround` later, I prefer to not touch here for now